src/controls/listView/IListView.ts

```diff
@@ -1,5 +1,7 @@
-import { Selection, SelectionMode } from 'office-ui-fabric-react/lib/DetailsList';
-import { IColumn, IGroup } from 'office-ui-fabric-react/lib/components/DetailsList';
+import {
+  SelectionMode, IColumn,
+  IDetailsRowProps, IGroup
```

Comment:
@milanholemans for consistency with the rest of the solution, can you please update the import to be more specific line the previous version? example: 
import { IColumn, IGroup } from 'office-ui-fabric-react/lib/components/DetailsList';